tests/osgi-tests/src/test/java/org/wso2/extension/siddhi/io/http/test/osgi/util/ServerUtil.java

```diff
@@ -0,0 +1,62 @@
+package org.wso2.extension.siddhi.io.http.test.osgi.util;
```

Comment:
Add licence

Reply:
Same applies to classes below

Reply:
Fixed in 6bf095af9d7a4c00220069badb50517cec553139